WooCommerce/WooCommerceTests/ViewRelated/Orders/Order Creation/ProductSelectorViewModelTests.swift

```diff
@@ -470,7 +511,9 @@ final class ProductSelectorViewModelTests: XCTestCase {
         viewModel.completeMultipleSelection()
 
         // Then
-        XCTAssertEqual(selectedItems, [simpleProduct.productID, 12])
+        if !ServiceLocator.featureFlagService.isFeatureFlagEnabled(.productMultiSelectionM1) {
```

Comment:
Please also replace this check and inject value for `shouldToggleItemOnSelection` in the initializer instead.

Reply:
Changed: https://github.com/woocommerce/woocommerce-ios/commit/82dfbb86c6d893a8526ce449998d7b438dc5c5c4